src/components/WalletBackupForm/WalletBackupForm.js

```diff
@@ -75,13 +76,12 @@ export function WalletBackupForm({
       />
       <h2 className={walletBackupFormStyle.title}>{t`Back Up ${name}`}</h2>
       <p className={walletBackupFormStyle.text}>
-        {hasMnemonicAttributes
-          ? getMnemonicAttributesFirstLine(passphrase, derivationPath)
-          : BACKUP_KEY_FIRST_LINE
-        }
-      </p>
-      <p className={walletBackupFormStyle.text}>
-        {hasMnemonicAttributes ? BACKUP_MNEMONIC_SECOND_LINE : BACKUP_KEY_SECOND_LINE}
+        {backupText.map((text: string, index: number) => (
```

Comment:
Maybe `.split('\n').join('<br />')` and `dangerouslySetInnerHTML` would look simpler?